src/app/main-overview/components/ZoneControlPlanesDetails.vue

```diff
@@ -1,12 +1,17 @@
 <template>
-  <KTable
+  <AppCollection
+    class="zone-cp-preview-collection"
+    data-testid="zone-cp-preview-collection"
     :headers="[
       { label: t('main-overview.detail.zone_control_planes.table.name'), key: 'name'},
       { label: t('main-overview.detail.zone_control_planes.table.status'), key: 'status'},
     ]"
-    :fetcher="() => ({ data: tableData })"
-    disable-sorting
-    hide-pagination-when-optional
+    :items="tableData"
+    :total="tableData.length"
+    :empty-state-title="t('zone-cps.empty_state.title')"
+    :empty-state-message="env('KUMA_ZONE_CREATION_FLOW') === 'enabled' ? t('zone-cps.empty_state.message') : t('common.emptyState.message', { type: 'Zones' })"
+    :empty-state-cta-to="env('KUMA_ZONE_CREATION_FLOW') === 'enabled' ? { name: 'zone-create-view' } : undefined"
```

Comment:
Something seems odd I have a "create zone button" when using Kuma:

<img width="390" alt="image" src="https://github.com/kumahq/kuma-gui/assets/702355/4895cdba-6638-4198-9e11-e497b190f188">

Also can we had an icon that is not Kong here? Clearly Kuma shouldn't show Kong logos.

Reply:
> Something seems odd I have a "create zone button" when using Kuma:

What’s odd here? This looks expected to me.

> Also can we had an icon that is not Kong here? Clearly Kuma shouldn't show Kong logos.

I’ve changed that for a more generic "no data" icon from the list here: https://kongponents.konghq.com/components/icon.html#state-icons